submissions/BlessingOkoro/ecommerce-project/phpunit.xml

```diff
@@ -0,0 +1,27 @@
+<phpunit
```

Comment:
while tests are very important, they are not the focus here, so this file can be deleted